libraries/eosiolib/contracts/eosio/map.hpp

```diff
@@ -216,29 +246,64 @@ namespace eosio::kv {
                itr_destroy(handle);
          }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief Utility function to see if the iterator is valid or is pointing 
+          * at `end` or one past the last or first element.
```

Comment:
@brief Utility function to see if the iterator is valid or is pointing 
          * at `end` or one past the last or first element.
would 
@ brief Utility function return true if iterator is valid else returns false if iterator is not valid - then define what is not valid
be better?

Reply:
true